### ISSUE TITLE
Meta files are executable in the release tarball

### ISSUE THREAD

**Issue by thaljef** [adamkennedy/PPI#167]
The files in git seems fine, but in the `MITHALDU/PPI-1.220.tar.gz` that I get from CPAN, the following files are all marked executable (0755):

```
Changes
LICENSE
MANIFEST
META.yml
Makefile.PL
README
README.md 
```

Was the release made on a Windows machine perhaps?

**Comment by wchristian** [comment]
Yep, thanks for the note, i'll look into that when i have some cycles. :)

**Comment by adamkennedy** [comment]
My release automation used to protect against that. It's possible they were
fucked up in the repo and I never fixed it because of that.

Adam
